### DISCUSSION TITLE
Info: DCS 2.7

### DISCUSSION THREAD

**Discussion by rkusa** [rkusa/DATIS#89] [General]
The latest versions of DATIS still work fine with DCS 2.7. I'll be looking into updating the weather part of the report to take the updated Cloud model into account.

**Comment by rkusa** [comment]
Initial support got added https://github.com/rkusa/DATIS/releases/tag/3.0.0-beta.1

**Comment by Colmill0** [comment]
Hi rkusa, thank you very much for your work. 
Since 2.7 I cannot get to work DATIS again, I followed your exactly instructions, and nothing...
Even I had to copy this :
dofile(lfs.writedir().."Mods\\services\\DATIS\\bin\\?.dll;")
to my \Saved Games\DCS.openbeta\Scripts\export.lua. I added it to the last line.
(Someone suggests to do it in DCS forum)

I am noticed that in SR Client list now appear other name (not my callsign):
PILOT_135657
For a micro seconds appear mi correct callsign, and then changes to these.
Maybe that is the problem, can you help me?

Thanks in advance

**Reply by rkusa** [reply]
Hi, 

```
dofile(lfs.writedir().."Mods\services\DATIS\bin\?.dll;")
```

This shouldn't be necessary.

Can you set the [`debug` setting](https://github.com/rkusa/DATIS/blob/master/docs/Settings.md) to `true`, try again and send me the log file (`Saved Games\DCS.openbeta\Logs\DATIS.log`)?

**Comment by Colmill0** [comment]
Hi Markus, and thank you for answer:
[DATIS.log](https://github.com/rkusa/DATIS/files/6602738/DATIS.log)
![3](https://user-images.githubusercontent.com/85252541/120896173-5c846180-c5ee-11eb-99a2-43fb389ba204.png)

While doing what you told me, I realized that the problem was "Secure Coalition Radio ON"
I changed it to OFF and it works perfect.

But how can I prevent Blue and Red from hearing the same thing?

Thank you

**Reply by rkusa** [reply]
Thanks for the update. I've created an issue to track the bug with Secure Coalition Radio being enabled (https://github.com/rkusa/DATIS/issues/98).
Until it is fixed, you could use radio encryption to prevent your enemies from listening to your radio calls 😊

**Reply by Colmill0** [reply]
Thanks Markus, your work is very valuable to us.
Cheers

**Comment by Colmill0** [comment]
Hello again

It seems to me that with the last (perhaps penultimate) update, I have not managed to make the DATIS work in the Syrian and Mariana scenarios.
I attach the missions (of all the maps) and the log.
Watch for your comments

Regards

[ATIS All Maps.zip](https://github.com/rkusa/DATIS/files/6765945/ATIS.All.Maps.zip)
[DATIS.log](https://github.com/rkusa/DATIS/files/6765946/DATIS.log)

**Reply by rkusa** [reply]
Hi, I think you just have to upgrade to the latest version `3.1.2`, which includes some fixes for airfields on Syria and Mariana (you are using `3.1.0` according to the logs). 😊

**Reply by Colmill0** [reply]
You are absolutely right...
Thanks!

**Comment by Colmill0** [comment]
HELLO

I am trying to transmit the ATIS on 2 frequencies (UHF and VHF), I have not managed to do it.
It's possible?

I have tried this: (also statics)

ATIS Incirlik 129.650, TRAFFIC 123.000, NO QFE
ATIS Incirlik 235.000, TRAFFIC 313.600, NO QFE

Thanks

**Reply by rkusa** [reply]
Hi, DATIS currently only supports one station per airfield. This is a restriction I'll probably lift eventually.

**Reply by Colmill0** [reply]
Thanks anyway, I imagine it must not be easy to achieve something like that.

Cheers
